doc/source/train/user-guides/persistent-storage.rst

```diff
@@ -378,16 +444,13 @@ Note that this behavior is off by default.
             # Every local rank 0 worker saves artifacts.
             ...
 
-
-.. _train-storage-advanced:
-
-Advanced configuration
-----------------------
-
 .. _train-local-staging-dir:
 
-Setting the local staging directory
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+(Deprecated) Setting the local staging directory
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. note::
+    This section describes behavior depending on Ray Tune implementation details that no longer applies to Ray Train V2.
```

Comment:
Doesn't this still apply to Train V2?

Reply:
There's no more concept of the `driver_artifacts` folder, but you're right, the working directories of each worker still gets set to a folder in the ray session directory. But Ray Train doesn't write stuff into that, only the user does.

I can move this back and remove references to the driver artifacts.